tests/damped_viscoelastic_bending_beam.prm

```diff
@@ -0,0 +1,49 @@
+# This test checks whether the viscoelastic bending beam benchmark
+# run successfully. In particular, this benchmark is a good test to
+# ensure that a model with compositional fields that affect material
+# properties works.
```

Comment:
For simplicity, I think you could remove the two sentences after the sentence that contains "run successfully".